account-management-api/src/main/java/uk/gov/di/accountmanagement/lambda/UpdateEmailHandler.java

```diff
@@ -165,9 +166,9 @@ public APIGatewayProxyResponseEvent updateEmailRequestHandler(
 
             auditService.submitAuditEvent(
                     AccountManagementAuditableEvent.UPDATE_EMAIL,
-                    AuditService.UNKNOWN,
+                    ClientSessionIdHelper.extractSessionIdFromHeaders(input.getHeaders()),
                     sessionId,
-                    AuditService.UNKNOWN,
+                    input.getRequestContext().getAuthorizer().get("clientId").toString(),
```

Comment:
If the authorizer params do not contain "clientId" this will throw a null pointer error. I don't know enough about whether we can guarantee that "clientId" will always be present, but I think we should build this resilience in - maybe rather than `.get("clientId")` we could do `getOrDefault("clientId", "unknown").toString()